modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -428,10 +428,15 @@ inline const char* _opencv_avcodec_get_name(CV_CODEC_ID id)
 }
 
 
-static
-inline int _opencv_ffmpeg_interrupt_callback(void *ptr)
+static int _opencv_ffmpeg_interrupt_callback(void *ptr)
 {
     AVInterruptCallbackMetadata* metadata = (AVInterruptCallbackMetadata*)ptr;
+
+    if(!metadata)
+    {
+        CV_LOG_DEBUG(NULL, "Stream timeout without metadata passed");
+    }
```

Comment:
Both messages should be visible by default - `_WARNING()` is more applicable here than `_DEBUG()`

---

missing `return -1`?

Reply:
There is `CV_Assert(metadata);` later in the code. Looks like it's not expected case or assert is mistake. I'll try to debug.